eselect-repo-helper

```diff
@@ -48,7 +48,7 @@ def do_list(args):
             status = 'disabled'
         repos[name] = {
             'status': status,
-            'url': r.findtext('homepage'),
+            'url': str(r.findtext('homepage')).strip(),
```

Comment:
Hmm, why do you need `str()`?

Reply:
I guess, if `r.findtext` can not get a homepage a `NoneType object` has to be converted to string before stripping.

`'url': r.findtext('homepage').strip()` leads to
```
Traceback (most recent call last):
  File "/usr/lib/eselect-repo/eselect-repo-helper", line 162, in <module>
    main()
  File "/usr/lib/eselect-repo/eselect-repo-helper", line 152, in main
    do_list(args)
  File "/usr/lib/eselect-repo/eselect-repo-helper", line 51, in do_list
    'url': r.findtext('homepage').strip(),
AttributeError: 'NoneType' object has no attribute 'strip'
```

Reply:
You don't want to strip that, you want to pass `None` over. However, I don't think stripping solves the problem fully. We need to replace all whitespace, not just strip the edges. I see that the whole thing is much more broken than that, so I'll look into fixing it myself.

Reply:
well, an url with whitespace is not a valid url… however, yes this might not solve all problems but works fine for the current status.

Reply:
I see that I was wrong. It's not eselect-repo being broken but repositories.xml had invalid data. I'll fix this bug now.